src/adapter.js

```diff
@@ -300,15 +307,133 @@ var createRegExp = function (filter) {
   return new RegExp(patternExpression, patternSwitches)
 }
 
+function getGrepSpecsToRun (clientConfig, specs) {
+  var grepOption = getGrepOption(clientConfig.args)
+  if (grepOption) {
+    var regExp = createRegExp(grepOption)
+    return filter(specs, function specFilter (spec) {
+      return regExp.test(spec.getFullName())
+    })
+  }
+}
+
+function parseQueryParams (location) {
```

Comment:
why not just use `querystring.parse()`?
https://nodejs.org/api/querystring.html#querystring_querystring_parse_str_sep_eq_options

Reply:
Because the adapter runs in the browser.